data/ko/config.yaml

```
@@ -0,0 +1,165 @@
+- name: rules_file
+  type: List
+  description: |
+    규칙 파일(들)의 위치입니다. 여기에는 별도 규칙 파일에 대한 하나 이상의 경로가 포함될 수 있습니다. 다음의 예는 동일합니다.
```

```suggestion
    규칙 파일(들)의 위치이다. 여기에는 별도 규칙 파일에 대한 하나 이상의 경로가 포함될 수 있다. 다음의 예는 동일하다.
```
소개를 제외한 문서는 평어체로 진행하기로 했습니다.

---

반영하겠습니다.